docs/index.md

```diff
@@ -3,12 +3,146 @@
+
+The fields needed for running this workflow are shown on the page.
+We will be configuring the "Image file" field next.
+
+<img src="img/workflows_image_cc_fields.png" width="450" alt="Canopy Cover workflow fields" aria-label="Available Canopy Cover workflow fields" title="Canopy Cover workflow fields"/>
+
+Select the ellipses next to "Image file" to bring up the browsing window.
+Since we've  only defined one file configuration, it will be automatically chosen for us.
+Select the orthomosaic file and click "OK" to confirm your selection and dismiss the window.
+If you are using the sample data, the name of the file to choose is "sample_orthophoto.tif".
+
+<img src="img/workflows_image_cc_field_file.png" width="450" alt="Selecting image to process" aria-label="File selection window" title="Select image to process"/>
+
+Click the ellipses next to each of the other entries to choose the GeoJSON and YAML experiment files for this workflow.
+
+Once the fields are fill in, click the "Run" button under the entries to start the workflow.
+
+<img src="img/workflows_image_cc_start.png" width="450" alt="Starting the workflow" aria-label="Start the workflow" title="Starting the workflow"/>
+
+You will be returned to the Workflow listing page showing your running job.
+The status of the job is automatically updated.
+Other jobs may also be displayed in the listing.
+Next we are going to navigate to view the output of the running workflow.
+
+<img src="img/workflows_image_cc_listing.png" width="450" alt="Listing of workflows" aria-label="Listing of new workflow" title="Listing of workflows"/>
+
+To see how the workflow job is proceeding, click the "View" button next to the workflow we just created.
+This will display a page where the status of the job is display and you can access normal and warning/error messages.
+To update the messages and warnings/errors, click the "Refresh" button.
+If there are many messages retrieved from the server, it may take a few seconds before the messages are displayed.
+
+The current count of regular and warning/error messages are updated after refreshing.
+Clicking on the counts will switch between the message types.
+
+<img src="img/workflows_image_cc_messages.png" width="450" alt="Workflow output listing refresh" aria-label="Viewing output messages" title="Refresh view of output messages"/>
+
+Use the "back" button to return to the workflow listing page.
+Click the "View" button again on the workflow listing page to re-display the messages.
+
+<img src="img/workflows_image_cc_messages_back.png" width="450" alt="Back button to view workflows" aria-label="Click back button for workflow listing" title="Click the back button"/>
+
+### Downloading the results
+
+On the workflow listing page, completed workflows have a download button to the far right of their name.
+Clicking the download button will download the CSV file ccontaining the results and the JSON of the job definition.
```

Comment:
canopy_cover.csv looks great!

canopy_cover.json contains:

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

is there a way to download the clipped and masked images?

Reply:
Canopy cover JSON file should not be doing that; I'll look into it. There isn't a way to get everything downloaded - I'll put in an issue

Reply:
>  get everything downloaded 

It should still be an option to download the intermediate files - some people may just want the CSV files.

Reply:
Issues written up:
https://github.com/AgPipeline/issues-and-projects/issues/552
https://github.com/AgPipeline/issues-and-projects/issues/557